### PR TITLE
Adding a comms opt-in that should store the users preference in the DB

### PR DESCRIPTION
**Pull request by jon-betts**
* This currently has no effect
 * There is no DB migration yet, so this shouldn't be merged until then
 * Opt-ing out will be in a separate PR

For: https://github.com/hypothesis/product-backlog/issues/1086

### CODE REVIEW

File: h/accounts/schemas.py [hunk 1]
```diff
@@ -162,6 +162,16 @@ class RegisterSchema(CSRFSchema):
         ),
     )
 
+    comms_opt_in = colander.SchemaNode(
+        colander.Boolean(),
+        description=_("I would like to receive news about annotation and Hypothesis"),
```
Comment (seanh) [inline h/accounts/schemas.py:167]:
```suggestion
        description=_("I would like to receive news about annotation and Hypothesis."),
```

Screenshot from 2020-08-12 09-22-19

Comment (seanh) [reply h/accounts/schemas.py:167]: I wonder whether "receive news" should say "receive email" as well, just to be clear. Right now the form doesn't actually say that they're consenting for us to _email_ them

**Comment by seanh** [comment]
Added a note about account activation to the issue https://github.com/hypothesis/product-backlog/issues/1086#issuecomment-672736386

File: h/accounts/schemas.py (hunk 1 above)
Comment (jon-betts) [reply h/accounts/schemas.py:167]: The text here was lifted directly from: https://github.com/hypothesis/product-backlog/issues/1086 (lack of full stop and all). Although I could have probably spotted that one.

**Comment by jon-betts** [comment]
The migration has gone out now

**Pull request merged**